lava/lava-job-definitions/shared/templates/base.yaml

```diff
@@ -43,6 +43,19 @@ protocols:
     verbose: true
 {% endif %}
 
+{% if not root_partition %}
```

Comment:
I guess we can do better here and remove the if statements.
```
{%    set root_partition = root_partition|default(2, true)  %}
```

Reply:
changed